src/transform/class/PotentialClass.js

```diff
@@ -11,17 +13,20 @@ class PotentialClass {
    * @param {Object} cfg
    *   @param {String} cfg.name Class name
    *   @param {PotentialMethod} cfg.constructor
+   *   @param {String} cfg.superClass Super Class name
    *   @param {Object} cfg.fullNode Node to remove after converting to class
    *   @param {Object[]} cfg.commentNodes Nodes to extract comments from
    *   @param {Object} cfg.parent
    */
-  constructor({name, constructor, fullNode, commentNodes, parent}) {
+  constructor({name, constructor, superClass, fullNode, commentNodes, parent}) {
     this.name = name;
     this.constructor = constructor;
+    this.superClass = superClass;
```

Comment:
The `superClass` config is never passed to constructor. So this is dead code and better to be removed.